### PR TITLE
fix(openrtb): revert back OpenRTB extension

### PR DESCRIPTION
**Pull request by garukun**
# What

The new Extension usage is blocking development, let's revert first and revisit with a better solution.

### CODE REVIEW

File: openrtb/application.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ type Application struct {
 	IsPaid            NumericBool `json:"paid,omitempty"`
 	Publisher         *Publisher  `json:"publisher,omitempty"`
 	Keywords          string      `json:"keywords,omitempty"`
-	Extension         *Extension  `json:"ext,omitempty"`
```
Comment (tiagozortea) [inline openrtb/application.go]: Is the `Extension` type used for anything anymore? If no can you delete it?

File: openrtb/bidrequest.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ type BidRequest struct {
 	BlockedCategories  []Category    `json:"bcat,omitempty"`
 	BlockedAdvertisers []string      `json:"badv,omitempty"`
 	Regulation         *Regulation   `json:"regs,omitempty"`
-	Extension          *Extension    `json:"ext,omitempty"`
+	Extension          interface{}   `json:"ext,omitempty"`
```
Comment (hwong) [inline openrtb/bidrequest.go]: Remove this line, originally it wasn't there.

File: openrtb/deal.go [hunk 3]
```diff
@@ -11,5 +11,5 @@ type Deal struct {
 	BidFloorCurrency Currency    `json:"bidfloorcur,omitempty"`
 	AuctionType      AuctionType `json:"at,omitempty"`
 	WhitelistedSeats []string    `json:"wseat,omitempty"`
-	Extension        *Extension  `json:"ext,omitempty"`
+	Extension        interface{} `json:"ext,omitempty"`
```
Comment (hwong) [inline openrtb/deal.go]: Remove this line, we don't have any deal extensions yet.

File: openrtb/privatemarketplace.go [hunk 4]
```diff
@@ -8,5 +8,5 @@ package openrtb
 type PrivateMarketplace struct {
 	IsPrivateAuction NumericBool `json:"private_auction"`
 	Deals            []*Deal     `json:"deals"`
-	Extension        *Extension  `json:"ext,omitempty"`
+	Extension        interface{} `json:"ext,omitempty"`
```
Comment (hwong) [inline openrtb/privatemarketplace.go]: Remove this line, we don't have any extensions yet.

File: openrtb/application.go (hunk 1 above)
Comment (garukun) [inline openrtb/application.go]: :trollface:

Comment (tiagozortea) [inline openrtb/application.go]: Nvm

**Comment by garukun** [comment]
PTAL

**Comment by hwong** [comment]
LGTM

**Comment by tiagozortea** [comment]
LGTM

**Pull request merged**